oshi-core/src/main/java/oshi/software/os/unix/freebsd/FreeBsdOSProcess.java

```diff
@@ -223,6 +226,30 @@ private int queryBitness() {
         return 0;
     }
 
+    @Override
+    public List<OSThread> getThreadDetails() {
+        List<OSThread> threads = new ArrayList<>();
+        String psCommand = "ps -awwxo tdname,lwp,state,etimes,systime,time,tdaddr,nivcsw,nvcsw,majflt -H";
```

Comment:
add `minflt` and `pri` to this.

Reply:
Man page of ps says `minflt	total page reclaims`. I am not sure if we should use that. Also we don't have any method for priority in the OSThread interface. Should we add priority for everything?

Reply:
I missed that we removed it.  Yes, we should have priority.  

Also minor page faults and reclaims are a synonym.  The very common paging with memory that’s not a big problem.  Major page faults (disk paging) are more serious.  Windows doesn’t distinguish between them,  it we should include both if possible.

Reply:
Have been on the road posting from phone. Summarizing:
1. Please add back in priority.  Driver for windows already has CURRENTPRIORITY field to include. Not available on mac. I assume linux has it in the proc/pid/task/tid/stat output. Hopefully you can include all that.
2. Please include minflt from unix, it has same meaning/different words as linux.   

Other than that it really looks good and hopefully Solaris with ps and/or prstat should be similar.

Reply:
Relax i am working on it and probably finish it this weekend. Although Some of the test cases are failing in freebsd `SensorTest.java` `testSensors` and `GlobalMemoryTest : Available memory should be less than or equal to total memory`. This may be because of the VM thing but i am no of sure.

Reply:
I’m on the road all weekend so take your time. :)

Reply:
I’m on the road all weekend so take your time. :)

Reply:
Re: FreeBSD memory, check the relevant sysctl calls

Reply:
Re: FreeBSD memory, check the relevant sysctl calls